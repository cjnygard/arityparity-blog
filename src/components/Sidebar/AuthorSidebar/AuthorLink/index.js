// @flow strict
export { default } from './AuthorLink';
