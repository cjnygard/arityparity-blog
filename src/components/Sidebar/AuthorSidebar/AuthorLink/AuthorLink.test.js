// @flow strict
import React from 'react';
import renderer from 'react-test-renderer';
import AuthorLink from './AuthorLink';

describe('AuthorLink', () => {
  const props = {
    author: {
      name: 'test',
      photo: '/carl-photo-social.jpg',
      bio: 'test'
    },
    isIndex: false
  };

  it('renders correctly', () => {
    const tree = renderer.create(<AuthorLink {...props} />).toJSON();
    expect(tree).toMatchSnapshot();
  });
});
