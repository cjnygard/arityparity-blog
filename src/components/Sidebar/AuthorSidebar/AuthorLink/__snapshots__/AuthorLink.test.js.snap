// Jest Snapshot v1, https://goo.gl/fbAQLP

exports[`AuthorLink renders correctly 1`] = `
<ul
  className="author"
>
  <li>
    <div
      className="author__headshot"
    >
      <img
        alt="test"
        className="author__headshot_photo"
        height="75"
        src="/carl-photo-social.jpg"
        width="75"
      />
    </div>
  </li>
  <li>
    <div
      className="author__name"
      height="75"
    >
      <a
        className="author__name-link"
        href="/pages/test/"
      >
        test
      </a>
    </div>
  </li>
</ul>
`;
